packages/gatsby-transformer-json/src/__tests__/gatsby-node.js

```diff
@@ -26,18 +26,18 @@ describe(`Process JSON nodes correctly`, () => {
     node.content = JSON.stringify(data)
 
     const createNode = jest.fn()
-    const updateNode = jest.fn()
-    const boundActionCreators = { createNode, updateNode }
+    const addChildNodeToParentNode = jest.fn()
```

Comment:
It already existed?

Noted on length. Yeah, I like really explicit APIs. I'll make it `addNodeToParent`.